htmd/builder/amber.py

```diff
@@ -758,8 +776,9 @@ def run(self, result=None):
         unittest.TestCase.run(self, result) # call superclass run method
 
     def tearDown(self):
-        if self.currentResult.wasSuccessful():
-            shutil.rmtree(self.testDir)
+        pass
```

Comment:
Was this intentional or just something you use for debugging and commited by mistake?

Reply:
It doesn't work as I expected it to. It removes the folder even on failure and I couldn't find out the reason so I just removed it for the moment since it complicates debugging a lot if you lose the files.